lib/route/src/Obelisk/Route.hs

```diff
@@ -853,4 +853,33 @@ unsafeRenderBackendRoute backendRouteEncoder =
   let Right enc = checkEncoder backendRouteEncoder
   in renderBackendRoute enc
 
+-- | Renders a full backend url, using the provided route prefix (typically the scheme and authority).
+-- This is typically retrived from the "route" configuration in Obelisk applications. The prefix is
+-- assumed not to have a trailing slash.
+renderBackendUrl
+  :: Text -- ^ Base application route
+  -> Encoder Identity Identity (R (Sum br a)) PageName
+  -> R br
+  -> Text
+renderBackendUrl prefix enc = (prefix <>) . renderBackendRoute enc
```

Comment:
Because this function just does `<>` for you and introduces an important *assumption* (no trailing slash) I'd vote to just remove it. Just using `<>` is no harder and makes the assumption far more obvious.